### PR TITLE
Custom channel fixes

### PR DESCRIPTION
**Pull request by rtibbles**
Makes custom channels activatable in new settings paradigm.

Sets root node of custom channel import to "root".

### CODE REVIEW

**Comment by rtibbles** [comment]
Fixes #4438 too...

File: kalite/topic_tools/settings.py [hunk 1]
```diff
@@ -12,16 +12,22 @@
 import os
 from django.conf import settings
 
+CHANNEL = getattr(settings, "CHANNEL", "khan")
```
Comment (MCGallaspy) [inline kalite/topic_tools/settings.py]: If the user doesn't specify CHANNEL in their settings module, could the following situation be problematic?

```python
from django.conf import settings
from kalite.topic_tools import settings as tt_settings

settings.CHANNEL  # AttributeError
tt_settings.CHANNEL  # "khan"
```

**Comment by MCGallaspy** [comment]
Looks good, just one question.

File: kalite/topic_tools/settings.py (hunk 1 above)
Comment (rtibbles) [inline kalite/topic_tools/settings.py]: Yes, but I explicitly changed all references to CHANNEL to only come from
topic tools settings.

**Comment by MCGallaspy** [comment]
Still not working as I expected. Ran the same import command (`python bin\kalite manage contentload --import=D:\ka-lite\import_test --channel=test --data=D:\ka-lite\import_test`), no js console errors, but the sidebar is not responsive. I see the root node ("blah") but clicking it has no effect though the url is changed.

Navigating directly to the content item (`/learn/test/blah/astro/`) does indeed bring it up, but the sidebar is still wonky.

**Comment by MCGallaspy** [comment]
That said, the PDF reader works without a hitch!

**Comment by jamalex** [comment]
> That said, the PDF reader works without a hitch!

Note that there are two modes -- one in which we use pdf.js and one in which we launch an external viewer (for tablet clients, where pdf.js is too slow). We need to test that both modes work -- I'm worried that `settings.PDFJS = False` may no longer do anything, as I don't see a code path that references this, at first peek. We'll need this, for the Aflatoun deployment, as they're using tablets.

**Comment by MCGallaspy** [comment]
@jamalex this is diverging from the PR, but is there any documentation about pdf content? Like, how it can be configured, and what the expected behavior is for either mode?

**Comment by jamalex** [comment]
Not that I know of, no -- but the basic behavior is meant to be:

- `PDFJS = True` (default): Load the pdf.js library and use it to render the document inline within the page.
- `PDFJS = False`: Don't bother loading all the pdf.js libraries (they're big), and don't dare attempt to render in the browser -- just have an link to the PDF itself so that it will launch in an external viewer.

**Comment by rtibbles** [comment]
```python bin\kalite manage contentload --import=D:\ka-lite\import_test\blah --channel=test --data=D:\ka-lite\import_test```

Is the correct command, the import path should always reference the root node, not the containing folder.

**Comment by MCGallaspy** [comment]
@richard what's the difference between a root node and the base folder here? If I wanted everything in D:\ka-lite\import_test to become a top level topic/sidebar item, how could I do that?

**Comment by rtibbles** [comment]
Then you would target import_test, but you have the channel metadata in
there, so I assume you do not want to do that.

**Comment by MCGallaspy** [comment]
So is it forbidden to include the channel metadata there? Could it cause the sidebar problems I've been experiencing?

**Comment by MCGallaspy** [comment]
It's probably worth testing out channel import more extensively if we wish to make it user-facing eventually, or have partners run the command. This PR looks good to me, though.

**Comment by rtibbles** [comment]
I don't think we ever plan to make this user facing, as it should be
deprecated by content-curation.

**Pull request merged**